.github/workflows/test.yml

```diff
@@ -203,7 +211,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [macOS-latest, windows-2019, ubuntu-20.04]
-        ros_distribution: [foxy, galactic, rolling]
+        ros_distribution: [foxy, galactic, humble, rolling]
```

Comment:
humble and rolling should run on ubuntu-22.04, instead of ubuntu-20.04 on the line above.
Also,  steps

* ``Test single package with Connext dependency, default options``
* ``Check that rmw_implementation install directory is present``

check if ``matrix.os=='ubuntu-20.04'``, this needs to be taken care of too.

Reply:
Good catches! I fixed this in 61de228603907969a9c6b9751484782eb2d25646. I think using `exclude` is the simplest solution here.

Reply:
Looks good.